db/seeds.rb

```diff
@@ -123,3 +123,17 @@
   bar.increment!
 end
 puts "Created #{Booking.count} bookings..."
+
+puts "Creating 2 reviews for each experience"
+Experience.find_each do |experience|
+  users = User.all.sample(2)
+  users.each do |user|
+    review = Review.new(
+      user: user,
+      content: Faker::Restaurant.review,
+      experience: experience,
+      rating: rand(1..5)
+    )
+ review.save!
+  end
+end
```

Comment:
PR #42 already added review seeds. Did you branch off the latest `main`?

Reply:
pushed a new commit with edits:
 - deleted my seeds
 - changed avatar on reviews
 - changed render for reviews @trouni